Source/WebCore/platform/MediaSample.h

```diff
@@ -218,6 +219,8 @@ class MediaSamplesBlock {
         MediaTime duration { MediaTime::zeroTime() };
         std::pair<MediaTime, MediaTime> trimInterval { MediaTime::zeroTime(), MediaTime::zeroTime() };
         MediaSampleDataType data;
+        MediaSampleDataType hdrMetadata { nullptr };
```

Comment:
We may want this to be a SharedBuffer instead of a FragmentedSharedBuffer and make it contiguous upon demuxing considering it's pretty small to start.